src/server/models/db/migrations/20230725094906_add_email_type_columns_to_tables/migration.sql

```diff
@@ -1,46 +1,66 @@
-BEGIN;
```

Comment:
everything in here runs in a transaction already so begin is redundant (I don't think psql has nested transactions so I'm not sure how this worked)

Reply:
🤷 I think I've seen it used in another migration on the project.

Reply:
ahhh. ok, ignore me then!